index.html

```diff
@@ -31,6 +33,136 @@ <h2 class="connect">LET'S CONNECT</h2>
             <li><a href="https://www.twitter.com"><img src="images/angellist.png" /></a></li>
         </ul>
     </section>
+
+    <section class="work-section">
+        <article class="works">
+            <img src="./images/snapshoot-portfolio.png" class="snapshot"/>
+    
+            <h3 class="tonic">Tonic</h3>
+    
+            <ul class="tags">
+                <li class="tags-li-blod">Canopy</li>
+                <li class="tags-li">Back End Dev</li>
+                <li class="tags-li">2023</li>
+            </ul>
+    
+            <p class="pg2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="stack">
+                <li><button class="stack-elem">HTML</button></li>
+                <li><button class="stack-elem">CSS</button></li>
+                <li><button class="stack-elem">Javascript</button></li>
+            </ul>
+           <button class="see-project">See project</button>
+            
+    
+        </article>
+        
+
+        <article class="works">
+            <img src="./images/Snapshoot Portfolio2.png" class="snapshot"/>
+    
+            <h3 class="tonic">Multi-Post
+                Stories</h3>
+    
+            <ul class="tags">
+                <li class="tags-li-blod">Canopy</li>
+                <li class="tags-li">Back End Dev</li>
+                <li class="tags-li">2023</li>
+            </ul>
+    
+            <p class="pg2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="stack">
+                <li><button class="stack-elem">HTML</button></li>
+                <li><button class="stack-elem">CSS</button></li>
+                <li><button class="stack-elem">Javascript</button></li>
+            </ul>
+           <button class="see-project">See project</button>
+            
+    
+        </article>
+        
+    
+        <article class="works">
+            <img src="./images/Snapshoot Portfolio3.png" class="snapshot"/>
+    
+            <h3 class="tonic">Tonic</h3>
+    
+            <ul class="tags">
+                <li class="tags-li-blod">Canopy</li>
+                <li class="tags-li">Back End Dev</li>
+                <li class="tags-li">2023</li>
+            </ul>
+    
+            <p class="pg2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="stack">
+                <li><button class="stack-elem">HTML</button></li>
+                <li><button class="stack-elem">CSS</button></li>
+                <li><button class="stack-elem">Javascript</button></li>
+            </ul>
```

Comment:
- [ ] Nice work so far 👍🏿 but i notice you are using a` button `inside the `ul li` i kidly suggest you remove the `button `, as per project requirement it should just be a `ul li`
 - [ ] I also notice your design dose not aling with the figmaa design

 - [ ] your design
 
![image](https://github.com/umarahgul/PortfolioMicroverse/assets/68823331/fca90e1d-68b4-45ac-a342-57de303f92f5)

 - [ ] Figm Design

![image](https://github.com/umarahgul/PortfolioMicroverse/assets/68823331/65457cdc-258d-4efa-a6c1-dc77ae32796c)

- [ ] I kindly suggest you stick onto the figms drsign  provided 😉